### PR TITLE
FHV Player Cards

### PR DESCRIPTION
**Pull request by zzorba**
All FHV cards are present now and have been reviewed by @5argon.

The state release date is confusing, but FFG said in their livestream that it was Feb 16, 2024

### CODE REVIEW

**Comment by Hounk** [comment]
I was checking, if some cards were still missing and found out, the data is completed. However, by skimming over the code, I noticed a typo: "Dawn Star" is on position 131, not 132.

**Comment by 5argon** [comment]
@zzorba I opened PR on your branch with some more edits

File: pack/fhv/fhvp.json [hunk 1]
```diff
@@ -1302,57 +2204,163 @@
...
+    "name": "Mariner's Compass",
+    "pack_code": "fhvp",
+    "position": 122,
+    "quantity": 2,
+    "skill_intellect": 2,
+    "slot": "Hand",
+    "text": "[action] Exhaust Mariner's Compass: <b>Investigate.</b> If you fail, ready Mariner's Compass. If you succeed and have no resources in your resource pool, discover 1 additional clue at your location.\n[fast] During an investigation using Mariner's Compass, spend 1 resource: You get +1 [intellect] for this skill test. (Limit three times per investigation.)",
+    "traits": "Item. Tool.",
+    "type_code": "asset",
+    "xp": 0
```
Comment (fspoettel) [inline pack/fhv/fhvp.json:2278]:
```suggestion
    "xp": 2
```

File: pack/fhv/fhvp.json [hunk 2]
```diff
@@ -798,93 +1109,254 @@
...
+    "flavor": "\"No names, Skids.\"\n\"Sorry, Finn.\"",
+    "illustrator": "Lin Hsiang",
+    "name": "Bank Job",
+    "pack_code": "fhvp",
+    "position": 69,
+    "quantity": 2,
+    "skill_agility": 1,
+    "skill_combat": 1,
+    "skill_intellect": 1,
+    "slot": "Accessory",
```
Comment (derornos) [inline pack/fhv/fhvp.json:1353]: This card does not have a slot.

File: pack/fhv/fhvp.json [hunk 3]
```diff
@@ -798,93 +1109,254 @@
...
+    "subname": "From Distant Shores",
+    "text": "As an additional cost to play Bank Job, spend an action.\nInvestigators at your location gain a total of 8 resources, distributed as you wish.",
+    "traits": "Gambit. Double. Illicit",
+    "type_code": "asset",
```
Comment (derornos) [inline pack/fhv/fhvp.json:1357]: The type of this card is Event.

File: pack/fhv/fhvp.json [hunk 4]
```diff
@@ -601,7 +657,150 @@
...
+    "traits": "Spirit. Bold.",
+    "type_code": "event",
+    "xp": 2
+  },
+  {
+    "code": "10033",
+    "cost": 3,
+    "deck_limit": 2,
+    "faction_code": "guardian",
+    "flavor": "\"It cleans! It fixes! It picks, plucks, shines, and swishes!\"\n-Schoffner's Catalogue, Winter 1924 ed.",
```
Comment (derornos) [inline pack/fhv/fhvp.json:684]: This card does not have flavor text.

File: pack/dwl/wda.json [hunk 5]
```diff
@@ -226,6 +226,7 @@
         "sanity": 1,
         "skill_agility": 1,
         "slot": "Body",
+        "tags": "pa.",
```
Comment (fspoettel) [inline pack/dwl/wda.json:229]: was it cleared up whether the parley access is limited to _**Parley.**_ actions or does it also apply to cards like this?

In this specific case, it does not make a functional difference though as `Neutral 0-5` grants access anyway.

Comment (zzorba) [reply pack/dwl/wda.json:229]: I believe I got a clarification that it applies to parley generally -- though not on a card like Grizzled (which has Parley in italics as a rule reminder)

Comment (zzorba) [reply pack/dwl/wda.json:229]: As far as I know though, it doesn't actually matter yet (all the cards that mention parley are already rogue IIRC)

**Comment by robert-huang** [comment]
very minor typo, i opened a pr to your branch https://github.com/zzorba/arkhamdb-json-data/pull/4

**Comment by zzorba** [comment]
The announcement article has been posted, so these cards are now all officially out.

**Pull request merged**